Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/EditableRowSupport/EditModeController.cs

```diff
@@ -375,8 +386,11 @@ public void SynchronizeEditModeControls (BocColumnDefinition[] columns)
       if (!IsListEditModeActive)
         return new Tuple<BocListRow, EditableRow>[0];
 
+      Assertion.IsNotNull (_editedRowIDs, "_editedRowIDs must not be null.");
+      Assertion.IsNotNull (_editModeHost.Value, "_editModeHost.Value must not be null.");
+
       var availableRows = _editModeHost.Value.Cast<IBusinessObject>().Select ((o, i) => new BocListRow (i, o)).ToList();
-      var editedRows = _editedRowIDs.Select (rowID => _editModeHost.RowIDProvider.GetRowFromItemRowID (_editModeHost.Value, rowID)).ToList();
+      var editedRows = _editedRowIDs.Select (rowID => _editModeHost.RowIDProvider.GetRowFromItemRowID (_editModeHost.Value, rowID)!).ToList();
```

Comment:
there's some scary nullability stuff going on in this type and in EditableRow. maybe we should add an improvement ticket?